blivet/formats/fs.py

```diff
@@ -830,6 +831,7 @@ class Ext2FS(FS):
     _check = True
     _fsck_class = fsck.Ext2FSCK
     _info_class = fsinfo.Ext2FSInfo
+    _min_size = Size("2 MiB")
```

Comment:
This one is unneeded because of the one in `FS` above.

Reply:
Fixed.